umrr_can_publisher_FL:
  frame_id: "radar_FL"
  can_socket: "can1"
  debug: 0
  # parameters to send to sensor
  antenna_mode: {section: 3016, parameter: 0, value: 1}
  # center_frequency: {section: 3016, parameter: 5, value: 2}
